src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Square extends Quadrangle {
+
+    public Square(String color, int width, int height) {
+        super(color, width, height);
+    }
```

Comment:
Since `Square` is a specific type of quadrangle where all sides are equal, it should only require a single side length as a parameter, not width and height.